pages/scenarios/[id].vue

```diff
@@ -87,6 +87,60 @@
         </CAccordionItem>
       </CAccordion>
     </div>
+    <CAlert v-if="appStore.currentScenario.status.data?.runSuccess === false" color="danger">
+      The analysis run failed. Please
+      <NuxtLink prefetch-on="interaction" to="/scenarios/new">
+        try again.
+      </NuxtLink>
+      <p v-for="(errorMsg, index) in appStore.currentScenario.status.data.runErrors" :key="index">
+        {{ errorMsg }}
+      </p>
+    </CAlert>
+    <CRow v-else-if="appStore.timeSeriesData">
+      <div class="col-md-6">
+        <div class="card">
+          <div class="card-header border-bottom-0 d-flex justify-content-between">
+            <div class="d-flex align-items-center">
+              <CIcon icon="cilChartPie" size="xl" class="mb-1 text-secondary" />
+              <h2 class="fs-5 m-0 ms-3 chart-header">
+                Costs
+              </h2>
+            </div>
+          </div>
+          <div class="card-body">
+            <p>Placeholder for costs chart</p>
+          </div>
+        </div>
+      </div>
+      <div class="col-md-6">
+        <div class="card">
+          <div class="card-header border-bottom-0 d-flex justify-content-between">
+            <div class="d-flex align-items-center">
+              <CIcon icon="cilChartLine" size="xl" class="mb-1 text-secondary" />
+              <h2 class="fs-5 m-0 ms-3 chart-header">
+                Time series
+              </h2>
+            </div>
+            <TimeSeriesLegend />
+          </div>
+          <div
+            class="card-body p-0"
+            @mouseleave="onMouseLeaveTimeSeries"
+            @mouseover="() => { hideTimeSeriesTooltips = false }"
+          >
+            <TimeSeries
+              v-for="(_, seriesId, index) in appStore.timeSeriesData"
+              :key="seriesId"
+              :series-id="seriesId"
+              :index="index"
+              :opened-accordions="openedTimeSeriesAccordions"
```

Comment:
As suggested in `TimeSeries`, this could be something like
```suggestion
              :open="openedTimeSeriesAccordions.includes(seriesId)"
```